### DISCUSSION TITLE
<Your question or feedback about the GitHub Discussions product>

### DISCUSSION THREAD

**Discussion by coolboy9** [community/community#70485] [Programming Help]
### Select Topic Area

Question

### Body

I am an undergratude ,but i want to learn more and work in programming, how can I do? I am so nervious!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @coolboy9 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
